docker/aasp/Dockerfile.build

```diff
@@ -11,7 +11,7 @@ RUN cd cmd/aasp && CGO_ENABLED=0 GOOS=linux go build -o /aasp -ldflags="-s -w" m
 
 FROM alpine:3.18.4
 
-RUN apk update && apk --no-cache add curl
+RUN apk update && apk upgrade --no-cache && apk --no-cache add curl
```

Comment:
Is this resolved by just using a newer alpine image instead?

Reply:
3.18.4 is the newest version (which is why I thought the previous PR resolved it)